server/modules/instantiation/jobs.js

```diff
@@ -439,10 +446,6 @@ module.exports = function (config, portProvider, interpolationHelper, buildInsta
                 });
 
                 dockerCompose.on('exit', (code) => {
-                    reject(code);
-                });
-
-                dockerCompose.on('close', (code) => {
                     if (0 !== code) {
                         buildInstance.log('Failed to run docker-compose.');
                         reject(code);
```

Comment:
For now handling errors is really non-existent. But that something we should address in a separate PR after it is established how the information can be relayed to SPA (e.g. via websockets).